audio/pwaudio.c

```diff
@@ -14,26 +14,357 @@
 #define AUDIO_CAP "pipewire"
 #include "audio_int.h"
 
+typedef struct
```

Comment:
Why do you keep this struct anonymous? It is ok as it is `typedef`, but I feel it would be better to keep it consistent with the other structs in this file.

Reply:
I think that there's some confusion about what commit to review, it's better to squash everything to avoid this.